---
# Feel free to add content and custom Front Matter to this file.
# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults

layout: home
---

# Genepi Innsbruck


This site summarizes our main software projects. The focus of our projects is on mitochondrial DNA (mtDNA), Genotype Imputation, genome wide association studies (GWAS) and general frameworks for big data analysis.
{: .fs-6 .fw-300 }

For a list of all projects click [here](http://github.com/genepi/).

### mtDNA

|Project | About | Reference | Publication |
|-----|--------------|
| **Haplogrep** | HaploGrep is a tool for mtDNA haplogroup classification. |  <br> [https://github.com/seppinho/haplogrep-cmd](https://github.com/seppinho/haplogrep-cmd) <br> [https://haplogrep.i-med.ac.at/](https://haplogrep.i-med.ac.at/) | [Human Mutation 2010](https://onlinelibrary.wiley.com/doi/abs/10.1002/humu.21382) <br> [Nucleic Acids Research 2016](http://nar.oxfordjournals.org/content/early/2016/04/15/nar.gkw233) |
| **Haplocheck** | Haplocheck detects in-sample contamination in mtDNA or WGS sequencing studies by analyzing the mitchondrial content. | [https://github.com/genepi/haplocheck](https://github.com/genepi/haplocheck) <br> [https://mitoverse.i-med.ac.at/](https://mitoverse.i-med.ac.at/) | [Genome Research 2021](https://genome.cshlp.org/content/31/2/309) |
| **mutserve** | Mutserve is a variant caller for the mitochondrial genome to detect homoplasmic and heteroplasmic sites in sequence data. |  [https://github.com/seppinho/mutserve](https://github.com/seppinho/mutserve) <br> [https://mitoverse.i-med.ac.at/](https://mitoverse.i-med.ac.at/) | [Nucleic Acids Research 2016](http://nar.oxfordjournals.org/lookup/doi/10.1093/nar/gkw247) |

Please contact [Sebastian](mailto:sebastian.schoenherr@i-med.ac.at) or [Hansi](mailto:hansi.weissensteiner@i-med.ac.at) in case of questions.

### Imputation

|Project | About | Reference | Publication |
|-----|--------------|
| **Michigan Imputation Server** | Michigan Imputation Server provides a free genotype imputation service using Minimac4. | [https://github.com/genepi/imputationserver](https://github.com/genepi/imputationserver) <br> [https://imputationserver.sph.umich.edu](https://imputationserver.sph.umich.edu) |  [Nature Genetics 2016](https://www.nature.com/articles/ng.3656) |
|**Imputationbot**  |Automate interactions with Michigan Imputation Servers. |  [https://github.com/lukfor/imputationbot](https://github.com/lukfor/imputationbot) |

Please contact [Lukas](mailto:lukas.forer@i-med.ac.at) or [Sebastian](mailto:sebastian.schoenherr@i-med.ac.at) in case of questions.

### GWAS

|Project | About | Reference | Publication |
|-----|--------------|
|**PGS Calculation** | Applying polygenic scores (PGS) on imputed genotypes.| [https://github.com/lukfor/pgs-calc](https://github.com/lukfor/pgs-calc) |
|**GWAS-Regenie**  | A nextflow pipeline to perform whole genome regression modelling using regenie.      | [https://github.com/genepi/gwas-regenie](https://github.com/genepi/gwas-regenie) |

Please contact [Sebastian](mailto:sebastian.schoenherr@i-med.ac.at) or [Lukas](mailto:lukas.forer@i-med.ac.at) in case of questions.


### Frameworks

|Project | About | Reference | Publication |
|-----|--------------|
|**Cloudgene** | A framework to build Software As A Service (SaaS) platforms for data analysis pipelines.       | [https://github.com/genepi/cloudgene](https://github.com/genepi/cloudgene) | [BMC Bioinformatics](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-200) |
|**Cloudflow**  | Simplified pipeline creation for MapReduce and Spark. | [https://github.com/genepi/cloudflow](https://github.com/genepi/cloudflow) | [IEEE Conference 2015](https://ieeexplore.ieee.org/document/7160259) |
|**genepi-libs** | This repository includes libraries which are used in many of our scientific projects. | [https://github.com/genepi/genepi-libs](https://github.com/genepi/genepi-libs) |

Please contact [Lukas](mailto:lukas.forer@i-med.ac.at) or [Sebastian](mailto:sebastian.schoenherr@i-med.ac.at) in case of questions.

### Teaching

|Project | About | Reference | Publication |
|-----|--------------|
|**NGS-Class** | Introduction to bioinformatic analysis of next (NGS) and 3rd generation sequencing data.       | [https://github.com/genepi/ngs-class](https://github.com/genepi/ngs-class) |

Please contact [Stefan](mailto:stefan.coassin@i-med.ac.at) or [Sebastian](mailto:sebastian.schoenherr@i-med.ac.at) in case of questions.
